ts/src/base/Exchange.ts

```diff
@@ -3077,6 +3078,18 @@ export default class Exchange {
         throw new NotSupported (this.id + ' fetchPosition() is not supported yet');
     }
 
+    async fetchPositionsBySymbol (symbol: string, params = {}): Promise<any> {
```

Comment:
@ttodua does not need to be here, but I think it is time to add the `Position` type so we can type `fetchPositions, fetchPosition and fetchPositionsBySymbol`